trace-dispatcher/src/Cardano/Logging/Trace.hs

```diff
@@ -250,6 +251,30 @@ foldMTraceM cata initial (Trace tr) = do
         (lc, Left control) -> do
           T.traceWith tr (lc, Left control)
 
+-- | Like foldMTraceM, but filter the trace by a predicate.
```

Comment:
Can be removed, as a combination of foldMTraceM with filter can do the same. See above

Reply:
Again, explanation at the point of call.